src/utils/middleware.ts

```diff
@@ -34,8 +41,47 @@ const errorHandler = (error: Error, _req: Request, response: Response, _next: Ne
   }
 };
 
+/*
```

Comment:
One trick I learned was that if we start our comment above function with `/** ` then VSCode will automatically help us format the specs for the function. Not only that, we can see those specs whenever we hover over the function name when we call that function anywhere in the code.

Reply:
Wow! I didn't know that! That's incredibly useful. Will reformat my comments!